Kconfig

```diff
@@ -1919,6 +1926,20 @@ menu "LVGL configuration"
 			depends on LV_OS_WINDOWS
 			default n
 
+		config LV_USE_UEFI
+			bool "Use LVGL UEFI backend"
+			default n
```

Comment:
Nit: default option is disabled for bool type options in the kconfig, this can be removed.

Reply:
Didn't know that, thanks for the heads up.